alfalfa_worker/Dockerfile

```diff
@@ -76,34 +79,36 @@ RUN curl -SLO $ENERGYPLUS_DOWNLOAD_URL \
     && ln -s /usr/local/EnergyPlus-9-4-0/runenergyplus /usr/local/bin/ \
     && rm $ENERGYPLUS_DOWNLOAD_FILENAME
 
+ENV ENERGYPLUS_DIR /usr/local/EnergyPlus-9-4-0
```

Comment:
necessary?

Reply:
Probably not... I had originally added this to help `runenergyplus` find its executables. It does let the runenergyplus scritp get a little farther before erroring out, but since that issue is on Kyle's plate I didn't mean to let that slip through in this PR.